code/baseStation/lora_base/lora_base.ino

```diff
@@ -69,20 +62,29 @@ bool rcv_success = false;
 int packetID = 0;
 String lastPacket = "";
 uint8_t lastBuf[RH_RF95_MAX_MESSAGE_LEN];
-
+int csum = 0;
 
 String createPacket() {
   String p = "";
+  csum = 0;
   int len = Serial.available();
   if (!len){
-    p += "A";
+    p += "B";
     send_length = 1;
   } else {
     for (int i = 0; i < len; i++){
       p += (char)Serial.read();
     }
     send_length = len;
+    
+    if (send_length != 1) { //please excuse this
```

Comment:
just move it out to function with meaningful name and I wouldn't mind that at all! 👌

Reply:
Done and tested!